src/index.ts

```diff
@@ -94,6 +94,9 @@ const video = document.getElementById('video')
 if (/^(city|fantasy|forest|sea)$/.test(theme ?? '')) {
   addClass(video, `vjs-theme-${theme}`)
 }
+const {v: playbackId} = query
+const reportingUrl = `wss://sao-canary-catalyst-0.livepeer.fun/json_video+${playbackId}.js`
```

Comment:
The `v` query is not always a playback ID, it can be anything else that identifies a video like its IPFS CID. So we cannot use it to build the `reportingUrl`. You should have a logic the same way as before, parsing the video src URL to create the reporting URL, but doing it from the `getVideoSrc` response handler in the end of this file, which is where we get the playback URL (`src`).

Also for the future, is there a way for this reporting URL creation logic to be moved to the metrics library? I don't think users of our player data SDK should be handling this kind of formatting.